docs(autocomplete-theme-classic): rewrite docs (#465)
algolia/autocomplete: packages/website/docs/autocomplete-theme-classic.md

});
```

> If your renderer doesn't support `Fragment's, you can use `div.aa-ItemWrapper`.

Reviewer: ```suggestion
If your renderer doesn't support `Fragment`s, you can use `div.aa-ItemWrapper`.
```

Author: Grammarly fail.

});
```

## Additional CSS classes

The theme provides a set of optional classes for you to use in different contexts.

### Modifiers

- `.aa-ItemIcon--no-border` removes the border of the icon
- `.aa-ItemIcon--align-top` aligns the icon to the top (recommended when the template is longer than three lines)

> - `.aa-Panel--Scrollable` declares the scrollable container of the panel

Reviewer: ```suggestion
- `.aa-Panel--Scrollable` declares the scrollable container(s) of the panel
```
